.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ workflows:
 
       - run-task:
           name: backup environment
-          requires: [ api unit test, client unit test, integration test ]
+          requires: [ apply environment ]
```

Comment:
Nice, takes a minute and a half from the pipeline 👍

Reply:
Nope this was a dumb idea of mine. Lets approve yours :)